app/src/main/java/com/sun/livescore/util/Constants.kt

```diff
@@ -0,0 +1,7 @@
+package com.sun.livescore.util
+
+object Constants {
+    const val INVALID_ACTIVITY = "Invalid Activity"
+    const val START_CHAR = '0'
```

Comment:
tuơng tự

Reply:
biến `START_CHAR = '0'` để em dùng làm điều kiện trong vòng for ở function `getScoreFromString(scores: String?)` ở dưới ạ

Reply:
nếu chỉ dùng ở class nào thì tạo constant trong class đó